src/schemas/v1/calculator-endpoint.ts

```diff
@@ -37,14 +37,14 @@ const API_CALCULATOR_REQUEST_SCHEMA = {
     },
     utility: {
       type: 'string',
-      description:
-        'The ID of your utility company, as returned from /utilities. ' +
-        'Required if authority_types includes "utility".',
+      description: `The ID of your utility company, as returned from \
+\`/api/v1/utilities\`. Required if authority_types includes "utility". If \
+absent, no incentives offered by utilities will be returned.`,
     },
     items: {
       type: 'array',
-      description:
-        'Which types of product or project to fetch incentives for. If absent, include all products/projects.',
+      description: `Which types of product or service to fetch incentives for. \
+If absent, include incentives for all products and services.`,
```

Comment:
Do we need to expose a list of valid items that can be used here somewhere?

Reply:
Zuplo generates a list of values; it looks like this:

<img width="511" alt="Screenshot 2024-06-05 at 6 02 12 PM" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/107014/cbd51e30-c92d-452a-a14e-225b10b085f2">

Like I said in the commit message: because there are so many items, it would get way too long to include all of them here, and I'm planning to explain them on a separate page. I'll add a link to the separate page here (and in the `items` response field's docs) when it's ready.